data_model/src/expression.rs

```diff
@@ -305,9 +286,7 @@ impl From<Mod> for ExpressionBox {
 
 /// Evaluates to the right expression in power of left expressions.
 /// Works only for `Value::U32`
-#[derive(
-    Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Decode, Encode, Deserialize, Serialize, IntoSchema,
-)]
+#[derive(Debug, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema)]
 pub struct RaiseTo {
```

Comment:
I always found the inclusion of this operation baffling. I'm 100% confident we need to groom the expression API.